zos_concepts/templates/submit_job_template/runtime-requirements.yml

```diff
@@ -0,0 +1,31 @@
+################################################################################
+# Copyright (c) IBM Corporation 2023
+################################################################################
+
+################################################################################
+# List `requires_ansible_version` contains Ansible version entries that this
+# playbook requires. It can be one more then one Ansible version
+# e.g, `- 2.9` would be interpreted as 2.9 - 2.9.n where n is >= 0. Where `- 2.9.0`
+# would be require an exact match, just like 2.9.12 would be an exact match.
+#
+# Dictionary `requires_collection` can include more than one collection such as
+# in the case a playbook depends on other IBM z collections.
+# e.g,
+# requires_collection:
+#    - ibm_zos_core:
```

Comment:
Why does the documentation say requires 1.4.0, while the requirement below requires 1.7?

Reply:
I'd initially flagged this one too! However, the commented block is sample doc, the real block is further below
![image](https://github.com/IBM/z_ansible_collections_samples/assets/10567394/5761f002-0a9b-43d7-a3c7-568498d3a887)